Testscripts/Linux/utils.sh

```diff
@@ -1791,6 +1791,11 @@ function GetOSVersion {
                 os_CODENAME=${ver#*|}
                 os_RELEASE=${ver%|*}
                 os_UPDATE=${os_RELEASE##*.}
+                # For the scenario "7.0 Beta (Maipo)", only need to keep the number part for transfering int type
```

Comment:
transfering => transferring
Fix when os_UPDATE not only contains number, e.g. '7.0 Beta (Maipo)'

Reply:
Thank you for pointing out the mistake, fixed the notes according to your suggestion, please help to review again.
Thanks!